<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1282</width>
    <height>717</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Person Tracking System</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_display">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>1111</width>
      <height>601</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>image/t.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="line_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>1280</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>68</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(220,220,220)</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_title">
     <property name="geometry">
      <rect>
       <x>335</x>
       <y>19</y>
       <width>610</width>
       <height>35</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>35</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
font: 18pt &quot;Times New Roman&quot;;</string>
     </property>
     <property name="text">
      <string>Person Tracking System</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
    </widget>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>680</y>
      <width>1280</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>69</y>
      <width>20</width>
      <height>621</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>690</y>
      <width>1280</width>
      <height>180</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(233, 239, 251);</string>
    </property>
    <property name="title">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>215</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>525</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="cancel">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>584</y>
      <width>130</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 80, 83);font: 75 12pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>STOP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>427</y>
      <width>130</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(85, 255, 127);
font: 75 12pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>RUN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="toolButton_camera">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>273</y>
      <width>130</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Camera</string>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>370</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="toolButton_video">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>115</y>
      <width>130</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Upload</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
